Makefile

```diff
@@ -0,0 +1,30 @@
+# Variables
+CC = gcc
+CFLAGS = -Wall
+OPTIMIZATON1 = -o1
```

Comment:
needs to be uppercase "O"s

Reply:
By far the biggest difference in exec time is between no optimization (O0) and some optimization (O1). However I did not notice much interesting between the two traces, so not sure if we should have O0?

Reply:
O3 does not change much for either. WDYT having from O0 to O2 for both? Too much?

Reply:
I pushed a commit on top, for the comments above.